HLTrigger/special/src/HLTRechitsToDigis.cc

```diff
@@ -140,14 +140,17 @@ HLTRechitsToDigis::produce(edm::Event& iEvent, edm::EventSetup const& setup)  {
     // loop over the collection of rechits and match to digis
     EcalRecHitCollection::const_iterator ituneEB;
     for (ituneEB = recHitsHandle->begin(); ituneEB != recHitsHandle->end(); ituneEB++) {
-      EcalRecHit hit = (*ituneEB);
-      outputEBDigiCollection->push_back( (*digisEB->find(hit.id())).id(), (*digisEB->find(hit.id())).begin() );
+      EcalRecHit    hit = (*ituneEB);
```

Comment:
could you change this to

`EcalRecHit const & hit = * ituneEB;`

to avoid the copy ?